dissect/etl/headers/event.py

```diff
@@ -31,22 +31,22 @@ def read_instance_info(data: bytes) -> OrderedDict[str, Any]:
 
 def read_stack_trace(data: bytes) -> OrderedDict[str, Any]:
     instance_info = c_etl.EVENT_HEADER_EXT_TYPE_STACK_TRACE32(data)
-    address_length = (len(data) - 8) // instance_info._sizes["Address"]
+    address_length = (len(data) - 8) // instance_info.sizes["Address"]
```

Comment:
Reviewer: Not sure if the original is correct, shouldn´t it be `// 8`?

Reply:
I don't think the original is correct, it can be `// len(c_etl.uint32)` or just `// 4` for short.